src/libcrun/container.c

```diff
@@ -692,7 +692,7 @@ container_init_setup (void *args, char *notify_socket,
       if (UNLIKELY (*exec_path == NULL))
         {
           if (errno == ENOENT)
-            return crun_make_error (err, errno, "executable file not found in $PATH");
+            return crun_make_error (err, errno, "executable file %q not found in $PATH", def->process->args[0]);
```

Comment:
`%q` doesn't seem to work.  Could you please change it to \`%s\` ?

Reply:
I could not remember if %q was supported in C.  Too many years since I have played with it.